### PR TITLE
New Blog post and updates for Sagemaker inference

### PR DESCRIPTION
**Pull request by philschmid**
# What does this PR?

This PR updates the existing partnership blog post with the new resources and adds a quote for inference. It also introduces a new Blog post for inference `Deploy Hugging Face models easily with Amazon SageMaker`

### CODE REVIEW

File: deploy-hugging-face-models-easily-with-amazon-sagemaker.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+After we run our request we can delete the endpoint again with.
+
+
+```python
+# delete endpoint
+predictor.delete_endpoint()
+```
+
+### **Deploy the model from pre-trained artifacts using the <code>HuggingFaceModel</code> class**
```
Comment (jeffboudier) [inline deploy-hugging-face-models-easily-with-amazon-sagemaker.md:178]:
```suggestion
### **Deploy the model from pre-trained checkpoints using the <code>HuggingFaceModel</code> class**
```

File: deploy-hugging-face-models-easily-with-amazon-sagemaker.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+predictor.delete_endpoint()
+```
+
+
+
+## **Deploy one of the 10,000+ Hugging Face Transformers to Amazon SageMaker for Inference**
+
+To deploy a model directly from the Hugging Face Model Hub to Amazon SageMaker, we need to define two environment variables when creating the `HuggingFaceModel`. We need to define:
+
+* HF_MODEL_ID: defines the model id, which will be automatically loaded from[ huggingface.co/models](http://huggingface.co/models) when creating or SageMaker Endpoint. The 🤗 Hub provides +10 000 models all available through this environment variable.
```
Comment (jeffboudier) [inline deploy-hugging-face-models-easily-with-amazon-sagemaker.md:222]:
```suggestion
* HF_MODEL_ID: defines the model id, which will be automatically loaded from[ huggingface.co/models](http://huggingface.co/models) when creating or SageMaker Endpoint. The 🤗 Hub provides 10,000+ models all available through this environment variable.
```

File: deploy-hugging-face-models-easily-with-amazon-sagemaker.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+
+# **FAQ 🎯**
+
+You can find the complete [Frequently Asked Questions](https://huggingface.co/docs/sagemaker/faq) in the [documentation](https://huggingface.co/docs/sagemaker/faq).
+
+_Q: Which models can I deploy for Inference?_
+
+A: You can deploy:
+* any 🤗 Transformers model trained in Amazon SageMaker, or other compatible platforms and that can accomodate the SageMaker Hosting design
+* any of the 10 000+ publicly available Transformer models from the Hugging Face[ Model Hub](https://huggingface.co/models), or
```
Comment (jeffboudier) [inline deploy-hugging-face-models-easily-with-amazon-sagemaker.md:278]:
```suggestion
* any of the 10,000+ publicly available Transformer models from the Hugging Face[ Model Hub](https://huggingface.co/models), or
```

File: the-partnership-amazon-sagemaker-and-hugging-face.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ As a first step to enable our common customers, Hugging Face and Amazon are intr
 
 To learn how to access and use the new Hugging Face DLCs with the Amazon SageMaker Python SDK, check out the guides and resources below.
 
+> _At the 8th of July we extended the Amazon SageMaker experience to inference. If you want to learn how you can [deploy Hugging Face models easily with Amazon SageMaker](https://huggingface.co/blog/deploy-hugging-face-models-easily-with-amazon-sagemaker) take a look at the [new blog post](https://huggingface.co/blog/deploy-hugging-face-models-easily-with-amazon-sagemaker) or at the [documentation](https://huggingface.co/docs/sagemaker/inference)._
```
Comment (jeffboudier) [inline the-partnership-amazon-sagemaker-and-hugging-face.md:20]:
```suggestion
> _On July 8th, 2021 we extended the Amazon SageMaker integration to add easy deployment and inference of Transformers models. If you want to learn how you can [deploy Hugging Face models easily with Amazon SageMaker](https://huggingface.co/blog/deploy-hugging-face-models-easily-with-amazon-sagemaker) take a look at the [new blog post](https://huggingface.co/blog/deploy-hugging-face-models-easily-with-amazon-sagemaker) and the [documentation](https://huggingface.co/docs/sagemaker/inference)._
```

File: the-partnership-amazon-sagemaker-and-hugging-face.md [hunk 5]
```diff
@@ -74,6 +82,8 @@ Below you can find all the important resources to all published blog posts, vide
 - [Distributed Training Data Parallelism Tensorflow](https://github.com/huggingface/notebooks/blob/master/sagemaker/07_tensorflow_distributed_training_data_parallelism/sagemaker-notebook.ipynb)
 - [Distributed Training Summarization](https://github.com/huggingface/notebooks/blob/master/sagemaker/08_distributed_summarization_bart_t5/sagemaker-notebook.ipynb)
 - [Image Classification with Vision Transformer](https://github.com/huggingface/notebooks/blob/master/sagemaker/09_image_classification_vision_transformer/sagemaker-notebook.ipynb)
+- [Deploy one of the 10 000+ Hugging Face Transformers to Amazon SageMaker for Inference](https://github.com/huggingface/notebooks/blob/master/sagemaker/11_deploy_model_from_hf_hub/deploy_transformer_model_from_hf_hub.ipynb)
```
Comment (jeffboudier) [inline the-partnership-amazon-sagemaker-and-hugging-face.md:85]:
```suggestion
- [Deploy one of the 10,000+ Hugging Face Transformers to Amazon SageMaker for Inference](https://github.com/huggingface/notebooks/blob/master/sagemaker/11_deploy_model_from_hf_hub/deploy_transformer_model_from_hf_hub.ipynb)
```

**Review by jeffboudier — APPROVED** [review]
Added a few minor suggestions

File: deploy-hugging-face-models-easily-with-amazon-sagemaker.md [hunk 6]
```diff
@@ -0,0 +1,313 @@
+---
+title: 'Deploy Hugging Face models easily with Amazon SageMaker'
+thumbnail: /blog/assets/17_the_partnership_amazon_sagemaker_and_hugging_face/thumbnail.png
+---
+
+<img src="/blog/assets/17_the_partnership_amazon_sagemaker_and_hugging_face/cover.png" alt="hugging-face-and-aws-logo" class="w-full">
+
```
Comment (julien-c) [inline deploy-hugging-face-models-easily-with-amazon-sagemaker.md:7]: please add button/link to edit blogpost on GitHub @philschmid

File: _blog.yml [hunk 7]
```diff
@@ -160,3 +160,7 @@
 - local: sentence-transformers-in-the-hub
   title: "Sentence Transformers in the 🤗 Hub"
   date: June 28, 2021
+
+- local: deploy-hugging-face-models-easily-with-amazon-sagemaker
```
Comment (julien-c) [inline _blog.yml:164]: a bit late to the party, but next time please favor a shorter url @philschmid

File: deploy-hugging-face-models-easily-with-amazon-sagemaker.md (hunk 6 above)
Comment (philschmid) [reply deploy-hugging-face-models-easily-with-amazon-sagemaker.md:7]: added in #124

**Pull request merged**